torch/nn/functional.py

```diff
@@ -2843,48 +2940,6 @@ def interpolate(input, size=None, scale_factor=None, mode='nearest', align_corne
                 interpolate, (input,), input, size=size, scale_factor=scale_factor,
                 mode=mode, align_corners=align_corners,
                 recompute_scale_factor=recompute_scale_factor)
-    from .modules.utils import _ntuple
-
-    def _check_size_scale_factor(dim):
-        if size is None and scale_factor is None:
-            raise ValueError('either size or scale_factor should be defined')
-        if size is not None and scale_factor is not None:
-            raise ValueError('only one of size or scale_factor should be defined')
```

Comment:
What happened to this error message? We should try to preserve the original behavior as much as possible

Reply:
i thought they were saying the same thing, but i guess they're slightly different. readded